src/_data/team.yml

```diff
@@ -721,3 +721,21 @@
   linkedin: john-rihana-861b27a6
   email: john.rihana@codurance.com
   office: london
+
+- id: domingo_espinosa
+  name: Domingo Espinosa
+  image: /assets/custom/img/team/domingo_espinosa.jpg
+  role: Apprentice
+  fullDescription:
+      <p>Domingo Espinosa is a software developer with 10 years of experience in different companies and sectors.</p>
+      <p>He is a Java specialist, always looking for continues improvement. He is a great team maker with a serene and warm personality.</p>
```

Comment:
I think is `continuous improvement`

Reply:
`team maker` sounds weird to me, does `team player` fit in what you wanted to say?